src/libraries/System.Private.CoreLib/src/System/UnreachableException.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
```

Comment:
@bartonjs, there was discussion on the issue after approval about namespace. Is this the final verdict?

Reply:
Marked the issue so it comes up for discussion again today.

Reply:
In review today we landed on System.Diagnostics.